examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -839,9 +839,9 @@ def main(args):
     if args.train_text_encoder:
         text_lora_attn_procs = {}
         for name, module in text_encoder.named_modules():
-            if any(x in name for x in TEXT_ENCODER_TARGET_MODULES):
+            if name.endswith(TEXT_ENCODER_ATTN_MODULE):
                 text_lora_attn_procs[name] = LoRAAttnProcessor(
-                    hidden_size=module.out_features, cross_attention_dim=None
+                    hidden_size=module.out_proj.out_features, cross_attention_dim=None
```

Comment:
Looks good to me!

However, does the following need to be changed since that LoRA layer mapping is now being changed?

https://github.com/huggingface/diffusers/blob/2f997f30ab660472561f5e1b5232d4f116315b1b/src/diffusers/loaders.py#L74

Reply:
Cc: @patrickvonplaten do we need to change the split keys here for the text encoder since https://github.com/huggingface/diffusers/pull/3505/files#diff-eca8763d65ac395bf286dd84b25abcf5e299a737508155270a648534d781e865R34

Reply:
Nice catch! The current code becomes a problem when the `remap_key()` function is called, so I've fixed it 160a4d3. However, it seems that the current test code and dreambooth script do not fall into the conditions where this `remap_key()` function is called, so I haven't been able to test it. 

As far as I understand, the intention here is that this part is called when `AttnProcsLayers` is directly loaded with `load_state_dict()`, but are there any use cases where this part is called?

Reply:
Thanks! Yeah I think so too. @patrickvonplaten could you confirm once?